test/commands/ledger/download.test.ts

```diff
@@ -27,7 +26,7 @@ describe('ledger:download', async () => {
       expect(req.headers.get('Authorization')).to.equal(`Bearer ${testToken}`)
       const path = new URL(req.url).pathname
       switch(path) {
-        case '/ledgers/download' :
+        case '/ledgers/download' : {
```

Comment:
Really? That looks so bizarre in a case statement!

Reply:
I know why I don't like it. The closure doesn't do anything. One still has to `break` out of the case. Typescript needs better method pattern matching or something.

Reply:
So the reason eslint wants this is a pretty specific reason, it's because `const` was used inside the body of the 'case'.

I don't *love* this syntax either but it is one of those bug-preventing things. You may have noticed I only did it for this `case`, but not for the others (because it's the only one that defines a new variable).

So I like it for its purpose, but I'm with you in that I also think it's ugly ;)

Here's the eslint docs on this rule: 

https://eslint.org/docs/latest/rules/no-case-declarations

We can still disable it though, let me know! I super don't care

Reply:
Thanks for digging into these rules.